pkg/network/setup/netstat.go

```diff
@@ -202,47 +204,19 @@ func newVMIIfaceStatusFromGuestAgentData(guestAgentInterface api.InterfaceStatus
 		IP:            guestAgentInterface.Ip,
 		IPs:           guestAgentInterface.IPs,
 		InterfaceName: guestAgentInterface.InterfaceName,
-		InfoSource:    guestAgentInterface.InfoSource,
 	}
 }
 
-// MergeAgentStatusesWithDomainData merge QEMU interfaces with agent interfaces
-func MergeAgentStatusesWithDomainData(domInterfaces []api.Interface, interfaceStatuses []api.InterfaceStatus) []api.InterfaceStatus {
+// mergeAgentStatusesWithDomainData merge QEMU interfaces with agent interfaces
```

Comment:
Why the merge method is still needed? Couldn't find a place using the `name` field that is added here.

EDIT: ignore, I see it is done in the next commit.